models/meshmodel/registry/registry.go

```diff
@@ -201,7 +201,39 @@ func (rm *RegistryManager) RegisterEntity(h Host, en Entity) error {
 		return nil
 	}
 }
+func (rm *RegistryManager) GetCountForHostAndType(db *gorm.DB, hostID uuid.UUID, typeName string) (int64, error) {
+	var count int64
+	err := db.Table("hosts").
+		Select("hosts.id").
+		Joins("LEFT JOIN registries ON hosts.id = registries.registrant_id").
+		Where("hosts.id = ? AND registries.type LIKE ?", hostID, "%"+typeName+"%").
```

Comment:
Since your function call [here](https://github.com/meshery/meshery/pull/9040/files#diff-53b7da6cb4fd0ed620df83501e2b6b09a0648c5a84e3a9d589c06e02837a7655R1255) is not searching the type then lets not use `LIKE`.
Will you directly use the equality operator?

Reply:
okay roger that